docs/source/docs_user/docs_user_data_catalogue.rst

```diff
@@ -17,7 +17,7 @@ Feature Types
     <script>
     function loadDataSources() {
     
-      let url = "https://cabd-web.azurewebsites.net/cabd-api/docs?options=types"
+      let url = "https://cabd-web.azurewebsites.net/cabd-api/docs?options=barriers, dams, modelled crossings, fishways, stream crossings, waterfalls"
```

Comment:
Catalogue is not showing modelled crossings or stream crossings. Need to use underscore instead of space.